tests/test_cassettes/s3/test_s3_tape_cassette.py

```diff
@@ -276,6 +276,21 @@ def test_fetch_recording_ids_by_category_and_limit(self):
 
         self.assertEqual(2, len(list(self.cassette.iter_recording_ids(category='test_operation1', limit=2))))
 
+    def test_fetch_recording_ids_by_limit_overall(self):
+        recording1 = self.cassette.create_new_recording('test_operation1')
+        recording1._id = \
```

Comment:
That is not so nice.
Better way is to patch datetime.today() return value using python mock.

Reply:
done